chromium_src/chrome/browser/net/stub_resolver_config_reader.cc

```diff
@@ -0,0 +1,102 @@
+/* Copyright 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/net/stub_resolver_config_reader.h"
+#include "brave/components/brave_vpn/buildflags/buildflags.h"
+#include "build/buildflag.h"
+#include "chrome/browser/net/secure_dns_config.h"
+#include "components/prefs/pref_service.h"
+#include "content/public/browser/network_service_instance.h"
+#include "net/dns/public/dns_over_https_config.h"
+#include "services/network/public/mojom/network_service.mojom.h"
+#include "third_party/abseil-cpp/absl/types/optional.h"
+
+#if BUILDFLAG(ENABLE_BRAVE_VPN) && BUILDFLAG(IS_WIN)
```

Comment:
once again, there should not be any dependencies on brave vpn in the config reader. This pref should be added to chrome/common/pref_names.h

Reply:
done